server/plugin.go

```diff
@@ -314,6 +378,47 @@ func (p *Plugin) getZoomUserWithToken(token *oauth2.Token) (*zoom.User, error) {
 	return &zoomUser, nil
 }
 
+func (p *Plugin) getZoomUserWithSuperUserToken(email string, token *oauth2.Token) (*zoom.User, error) {
```

Comment:
`getZoomUserWithSuperUserToken` and `getZoomUserWithToken` is basically the same code, can't we remove some duplicates?